src/graphics.cpp

```diff
@@ -3,6 +3,8 @@
 #include <SDL2/SDL.h>
 #include "graphics.h"
 
+
+//Dibuja el primer pixel y calcula la orientación de la imagen a partir de ese offset
```

Comment:
Could you please write in English all comments in this code file? Also, remember to allow a space between // and your actual comment.